### DISCUSSION TITLE
Allow mouse button with keyboard modifier

### DISCUSSION THREAD

**Discussion by Pemto** [AppGround-io/bluetooth-keyboard-and-mouse-support#232] [Ideas]
I don't see a way to do a (control) key with the left mouse button, for example.  I only see keyboard options, or mouse, no way to mix them.

Suggest you add at least options to mix shift, ctrl, alt, with left, right, and middle mouse buttons.

My use case is in chrome a control+left mouse button, will open a link in a new background tab.

**Comment by Staini** [comment]
Combinations of custom mouse and keyboard buttons would be great! Nice idea :-)

**Comment by Kaptensvart** [comment]
Middle mouse click opens in new tab and u can close tabs with middle klick, or Ctrl+w if you don't wanna hower over the tab, useful for ads that pop up on a new tab.

![ec856e47732b1345743ed3fefe0ad9b1](https://user-images.githubusercontent.com/107816771/174503417-b7a8f4b8-45b3-4fb5-9543-ad9646582b78.jpg)
